inst/include/Rcpp/sugar/sets.h

```diff
@@ -22,40 +20,17 @@
 #ifndef Rcpp__sugar__sets_h
 #define Rcpp__sugar__sets_h
 
-#if __cplusplus >= 201103L
-    #define RCPP_UNORDERED_SET std::unordered_set
-    #define RCPP_UNORDERED_MAP std::unordered_map
-
-    namespace std {
-        template<>
-        struct hash<Rcpp::String> {
-            std::size_t operator()(const Rcpp::String& key) const {
-                return pointer_hasher( key.get_sexp() ) ;
-            }
-            hash<SEXP> pointer_hasher ;
-        };
-    }
-
-#elif defined(HAS_TR1_UNORDERED_SET)
-    #define RCPP_UNORDERED_SET std::tr1::unordered_set
-    #define RCPP_UNORDERED_MAP std::tr1::unordered_map
+#define RCPP_UNORDERED_SET std::unordered_set
+#define RCPP_UNORDERED_MAP std::unordered_map
```

Comment:
Wasn't this defined before?

Reply:
I did not add this so maybe consult `git blame` ?  And/or mark yourself a PostIt Note to remove it later.